File: Android_Zone_Test/src/com/example/mylib_test/delegates/TextDelegates.java
package com.example.mylib_test.delegates;

import com.example.mylib_test.R;
import com.zone.adapter3.bean.Holder;
import com.zone.adapter3.bean.ViewDelegates;

/**
 * [2018] by Zone
 */

public class TextDelegates extends ViewDelegates<String> {

    @Override
    public void fillData(int i, String s, Holder holder) {
        holder.setText(R.id.id_num, s);
    }

    @Override
    public int getLayoutId() {
        return R.layout.item_recycler;
    }
}

File: Android_Zone_Test/src/com/example/mylib_test/delegates/TextDelegates.kt
package com.example.mylib_test.delegates

import com.example.mylib_test.R
import com.zone.adapter3.bean.Holder
import com.zone.adapter3.bean.ViewDelegates

/**
 *[2018/7/10] by Zone
 */
class TextDelegates : ViewDelegates<String>() {

    override fun getLayoutId(): Int = R.layout.item_recycler
    override fun fillData(i: Int, s: String?, holder: Holder<out Holder<*>>?) {
        holder!!.setText(R.id.id_num, s)
    }
}
